cmd/kpod/ps.go

```diff
@@ -64,6 +75,18 @@ type psJSONParams struct {
 	Labels           fields.Set          `json:"labels"`
 	Mounts           []specs.Mount       `json:"mounts"`
 	ContainerRunning bool                `json:"ctrRunning"`
+	Namespaces       namespace           `json:"namespace"`
```

Comment:
Does this always appear in the JSON output even if the flag isn't specified? I feel like you'd need an `omitempty` to convince it to not appear, but if it works without that I'm fine with it.

Reply:
So the fixed.